### PR TITLE
DAOS-7174 doc: document daos_perf and fix a few nits

### PR DESCRIPTION
**Pull request by johannlombardi**
Fix a few intending issues (e.g. in property tables and dfuse modes).
Add an example for daos_perf.
Add an example for IL monitoring.

doc-only: true

Signed-off-by: Johann Lombardi <johann.lombardi@intel.com>

### CODE REVIEW

File: doc/user/workflow.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ All steps passed.
 !!! note
     The command is executed in a development environment,
```
Comment (cdurf1) [inline doc/user/workflow.md:100]: An open line is rqd here to render the Note

File: doc/user/workflow.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ All steps passed.
...
     performance differences will vary, based on your system.
+
 !!! warning
     Smaller pools may show DER_NOSPACE(-1007): 'No space
```
Comment (cdurf1) [inline doc/user/workflow.md:104]: And a line here.

**Review by cdurf1 — APPROVED** [review]

**Review by jolivier23 — CHANGES_REQUESTED** [review]
Looks fine but I wonder if we should just move this to #6223 

So +1 but with request changes so you at least read this comment before landing

**Comment by johannlombardi** [comment]
@jolivier23 got it.

**Pull request merged**